firmware/boron-ins-fsm/src/consoleFunctions.cpp

```diff
@@ -21,13 +21,14 @@
 void setupConsoleFunctions(){
 
   //particle console function declarations, belongs in setup() as per docs
+  Particle.function("Force_Reset", force_reset); 
+  Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);   
   Particle.function("Change_Initial_Timer", initial_timer_set); 
   Particle.function("Change_Duration_Timer", duration_timer_set);     
   Particle.function("Change_Stillness_Timer", stillness_timer_set);  
   Particle.function("Change_INS_Threshold", ins_threshold_set);
-  Particle.function("Turn_Debugging_Publishes_On_Off", toggle_debugging_publishes);   
```

Comment:
Any particular reason why this line moved?

Reply:
Well I figured that since "Turn_Debugging_Publishes_On_Off" is used the most on the particle console, it would be nicer to just have it at the top of the list. I also inserted the force reset at the top, so it is just under the force update. I can move it back if you do not like it.
<img width="330" alt="Screen Shot 2022-06-27 at 10 34 46 AM" src="https://user-images.githubusercontent.com/105232387/176001363-dd9b824d-be8f-47e3-9c4b-22ca6731521a.png">

Reply:
Haha, no, that's great. I didn't realize that the order in the code changed the order in Particle. This makes sense to me. Thanks!